docs/example.md

```diff
@@ -6,8 +6,8 @@ A Tableau connector is a set of files that describe the UI elements needed to co
 
 Tableau connector files include:
 - __Manifest file__ that tells Tableau about the connector. Tableau uses the name from this file to add it to the list of available connectors in the UI.
-- __Tableau Custom Dialog file__ customizes the connection dialog as needed for this connection. (If the base is not set to "odbc" or "jdbc", then this is file may not be required.)
-- __Tableau Connection Resolver file__ creates the connection to your file using JavaScript files to define and make the connection. (If the base is not set to "odbc" or "jdbc", then this is file may not be required.)
+- __Tableau Custom Dialog file__ customizes the connection dialog as needed for this connection. This file is optional. If not created, it will be based off the superclass set in the Manifest file (manifest.xml) by default. However if the superclass is set to "odbc" or "jdbc", then this file is required.
+- __Tableau Connection Resolver file__ creates the connection to your file using JavaScript files to define and make the connection. This file is optional. If not created, it will be based off the superclass set in the Manifest file (manifest.xml) by default. However if the superclass is set to "odbc" or "jdbc", then this file is required.
```

Comment:
I think this is written backwards. 99% of people should be using odbc or jdbc, so in 99% of the people reading this will be required to have these files. However, as it stands the first thing they read is "This file is optional," full stop, and they may not read further. 

I actually think the original is clearer on this - this file _may not_ be required in certain, very specific situations. I would hesitate to change that narrative.

Reply:
I will rephrase it to keep the original narrative,

Reply:
Was thinking of something like  (If the superclass in Manifest file (manifest.xml) is not set to the value “odbc” or “jdbc”, then this is file may not be required and it will be based off the odbc/jdbc based superclass specified.